queries/timedmetadata.sql

```diff
@@ -3,7 +3,7 @@ SELECT
 tm.id,
 m.primary_episode_id
 FROM timedmetadata tm
-         LEFT JOIN mediaitems m on m.id = tm.mediaitem_id
+LEFT JOIN mediaitems m on m.id = tm.mediaitem_id OR (m.timedmetadata_from_asset AND m.asset_id = tm.asset_id)
```

Comment:
```suggestion
LEFT JOIN (mediaitems m on m.id = tm.mediaitem_id) OR (m.timedmetadata_from_asset AND m.asset_id = tm.asset_id)
```

Personal preference for readability. Feel free to ignore.